operate/client/src/App/Processes/ListView/DiagramPanel/DiagramHeader/index.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {observer} from 'mobx-react';
+import isNil from 'lodash/isNil';
+import {CopiableProcessID} from 'App/Processes/CopiableProcessID';
+import {ProcessOperations} from '../../ProcessOperations';
+import {Restricted} from 'modules/components/Restricted';
+import {processesStore} from 'modules/stores/processes/processes.list';
+import {PanelHeader, Dd, Dl, Dt} from './styled';
```

Comment:
What do you think about changing `Dd`, `Dl` and `Dt` to more descriptive names? I can infer, but without context of the feature it might make legibility a bit harder in the future.